lib/ansible/modules/cloud/amazon/ec2_vpc_nacl_facts.py

```diff
@@ -132,9 +148,9 @@ def list_ec2_vpc_nacls(connection, module):
             nacl['tags'] = boto3_tag_list_to_ansible_dict(nacl['tags'], 'key', 'value')
         if 'entries' in nacl:
             nacl['egress'] = [nacl_entry_to_list(entry) for entry in nacl['entries']
-                              if entry['rule_number'] != 32767 and entry['egress']]
```

Comment:
I'm not familiar with th reason behind this check - why change it to `<` from `!=`?

Reply:
The module only returns "user" rules (i.e. it doesn't return the default deny all rule that each NACL has) which is why rule # 32767 was ignored. Per the [AWS API docs](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_CreateNetworkAclEntry.html) valid user rules are from 1 to 32766 and anything from 32767 to 65535 is reserved. The IPv6 default deny rule is # 32768.

However it looks like I flubbed this and it should be `< 32767` or `<= 32766` not `< 32766` 

```
RuleNumber
The rule number for the entry (for example, 100). ACL entries are processed in ascending order by rule number.

Constraints: Positive integer from 1 to 32766. The range 32767 to 65535 is reserved for internal use.

Type: Integer

Required: Yes
```